index.html

```diff
@@ -2,7 +2,50 @@
 <html>
   <head>
     <title>Memory exercise</title>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="memoryOO.js"></script>
+    <style>
+      .container {
+        width: 5rem;
+        margin: 0 auto;
+      }
+
+     td {
+        width: 2rem;
+        height: 2rem;
+        text-align: center;
+        border: 1px solid;
+      }
+
+      td.is-removed {
+        visibility: hidden;
+      }
+
+      button {
+        margin-bottom: 1rem;
+      }
+    </style>
   </head>
+
   <body>
+
+  <section class="container">
+  	<table class="board">
+      <tr>
+        <td>A</td>
+        <td>A</td>
+      </tr>
+      <tr>
+        <td>C</td>
+        <td>B</td>
+      </tr>
+      <tr>
+        <td>B</td>
+        <td>C</td>
+      </tr>
```

Comment:
Makes it a lot easier to play if I can see the values of all the tiles :smirk: 

![screen shot 2015-11-17 at 8 06 15 am](https://cloud.githubusercontent.com/assets/86842/11212217/58455ee2-8d02-11e5-9825-e3b4837e94d1.png)